admin/controllers/pupil.js

```diff
@@ -229,7 +229,8 @@ const postEditPupil = async (req, res, next) => {
   }
 
   const trimAndUppercase = R.compose(R.toUpper, R.trim)
-
+  await pupilAgeReasonService.refreshPupilAgeReason(pupil['id'], req.body.ageReason, pupil['ageReason'])
```

Comment:
why not `pupil.id` as per other calls?  threw me for a second...

Reply:
@GuyHarwood it's the same thing.
Reason why I did this is because my linter complains for an unresolved variable when described as `pupil.id` which is a minor warning.

Reply:
yes its just an object indexer, i get that.  In other places you use `pupil.id`, just looking for consistency 👍

Reply:
@GuyHarwood updated for consistency